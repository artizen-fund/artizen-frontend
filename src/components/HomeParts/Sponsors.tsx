import styled from 'styled-components'
import { assetPath } from '@lib'
import { breakpoint, typography } from '@theme'
import { partners as copy } from '@copy/home'

const Sponsors = () => (
  <Wrapper>
    <Label>{copy.label}</Label>
    <Partners>
      {copy.partners.map(partner => (
        <Partner key={`partner-${partner}`}>
          <img src={assetPath(`/assets/partners/${partner}.svg`)} alt={partner} />
        </Partner>
      ))}
    </Partners>
  </Wrapper>
)

const Wrapper = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  align-items: center;
  gap: 15px;
`

const Label = styled.h2`
  ${typography.label.l1}
`

const Partners = styled.ul`
  display: contents;
`

const Partner = styled.li`
  img {
    max-height: 56px;
  }
`

export default Sponsors
