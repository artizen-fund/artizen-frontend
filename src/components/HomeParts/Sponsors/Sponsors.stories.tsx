import Sponsors from './'

export default {
  title: 'components/Sponsors',
  component: Sponsors,
  argTypes: {},
}

export const SponsorsComponent = (props: any) => <Sponsors {...props} />
