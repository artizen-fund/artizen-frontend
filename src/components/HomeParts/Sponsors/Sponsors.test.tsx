import renderer from 'react-test-renderer'
import Sponsors from './'

describe('Sponsors', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Sponsors />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
