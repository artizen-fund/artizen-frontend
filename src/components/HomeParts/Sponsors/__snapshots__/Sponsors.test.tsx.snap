// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sponsors renders correctly 1`] = `
<div
  className="sc-6a1f4088-0 ivdrwk"
>
  <div
    className="sc-6a1f4088-1 bVoqbh"
  />
  <div
    className="sc-6a1f4088-2 gRmqJR"
  />
</div>
`;
